### PR TITLE
User can search

### PR DESCRIPTION
**Pull request by BrianBozi**
still a work in process

### CODE REVIEW

**Comment by BrianBozi** [comment]
live-search-desktop (1)

File: client/pages/postForm.jsx [hunk 1]
```diff
@@ -13,9 +13,9 @@ export default class NewPost extends React.Component {
   }
 
   handleChange(event) {
-    this.setState({
-      description: event.target.value
-    });
+    // this.setState({
+    //   description: event.target.value
+    // });
   }
```
Comment (scbowler) [inline client/pages/postForm.jsx]: If this method is no longer needed please remove it

File: client/pages/profiles.jsx [hunk 2]
```diff
@@ -7,23 +7,41 @@ export default class Profiles extends React.Component {
...
     };
+
+    this.getPosts = this.getPosts.bind(this);
   }
 
+  // componentDidMount() {
+  //   fetch(`api/profile/${this.props.userId}`)
+  //     .then(res => res.json())
+  //     .then(posts => this.setState({ posts: posts }));
+  // }
```
Comment (scbowler) [inline client/pages/profiles.jsx]: Remove commented code

File: server/public/styles.css [hunk 3]
```diff
@@ -264,13 +273,106 @@ textarea {
...
+  height: 100%;
+
+  /* z-index: 2; */
+
+  /* left: 50%; */
+
+  /* transform: translateX(-50%); */
+
+  /* top: 1.5rem; */
+}
```
Comment (scbowler) [inline server/public/styles.css]: Remove all commented CSS rules throughout the whole file

**Review by scbowler — CHANGES_REQUESTED** [review]
Looks good!  Just a few cleanup items needed. Let us know if you have any questions!

**Review by scbowler — APPROVED** [review]
👍 👍

**Pull request merged**